.github/workflows/velox_docker.yml

```diff
@@ -84,31 +89,41 @@ jobs:
           path: ./cpp/build/releases
       - name: Setup java and maven
         run: |
-          apt-get update && \
-          apt-get install -y openjdk-8-jdk maven && \
+          if [[ "${{ matrix.java }}" == "17" ]]; then
+            apt-get update && \
+            apt-get install -y openjdk-17-jdk maven
+          else
+            apt-get update && \
+            apt-get install -y openjdk-8-jdk maven
+          fi
           apt remove openjdk-11* -y
       - name: Build for Spark ${{ matrix.spark }}
         run: |
           cd $GITHUB_WORKSPACE/ && \
-          mvn clean install -P${{ matrix.spark }} -Pbackends-velox -DskipTests
+          mvn clean install -P${{ matrix.spark }} -P${{ matrix.java }} -Pbackends-velox -DskipTests
       - name: Build and run TPCH/DS ${{ matrix.spark }}
         run: |
           cd $GITHUB_WORKSPACE/tools/gluten-it && \
-          mvn clean install -P${{ matrix.spark }} \
+          mvn clean install -P${{ matrix.spark }} -P${{ matrix.java }} \
           && GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
             --local --preset=velox --benchmark-type=h --error-on-memleak --off-heap-size=10g -s=1.0 --threads=16 --iterations=1 \
           && GLUTEN_IT_JVM_ARGS=-Xmx5G sbin/gluten-it.sh queries-compare \
             --local --preset=velox --benchmark-type=ds --error-on-memleak --off-heap-size=10g -s=1.0 --threads=16 --iterations=1
 
-
-  run-tpc-test-centos7:
+  run-tpc-test-centos:
     needs: build-native-lib
     strategy:
       fail-fast: false
       matrix:
+        os: ["centos:7", "centos:8"]
         spark: ["spark-3.2", "spark-3.3", "spark-3.4", "spark-3.5"]
+        java: ["jdk-8", "jdk-17"]
```

Comment:
Shouldn't GitHub have some limits on matrix config? It's like 2 * 4 * 2 = 16 jobs generated...

Reply:
I'm not sure about it.. But seems it can work

Reply:
https://infra.apache.org/github-actions-policy.html

I wonder we may need to follow this policy from Apache infra. But not sure about it. Though it suggests concurrency <= 20.

Do we really need for both centos 7 / 8 in this job? Or is it still enough if we set a individual job for JDK 17 with out matrix?

Reply:
yeah, we can reduce some matrix. How about only test centos8 + ubuntu20.04 ? 

Is it a good time to drop supporting Spark3.2 ?

Reply:
> yeah, we can reduce some matrix. How about only test centos8 + ubuntu20.04 ?

Sounds good if it's enough. Thanks.

> Is it a good time to drop supporting Spark3.2 ?

It was on the plan to stop supporting 3.2 but looks like it may require for some time or some discussions to take the action. cc @zhouyuan @weiting-chen may have more contexts on that.

Reply:
as far as I know spark3.2 is still active in some production env, so better to keep it

Reply:
OK, in this pr I only add JDK17 test for Spark3.3. Do you have other concern ?